ts/features/itwallet/common/components/ItwDiscoveryBanner.tsx

```diff
@@ -35,16 +41,38 @@ export const ItwDiscoveryBanner = ({
     [isVisible, isItwTrialActive, isItwValid, isItwEnabled]
   );
 
+  const trackBannerProperties = React.useMemo(
+    () => ({
+      banner_id: "itwDiscoveryBannerTestID", // Banner testID prop
+      banner_page:
+        navigation?.getState()?.routes?.[navigation?.getState()?.index]?.name,
+      banner_landing: "ITW_INTRO"
+    }),
+    [navigation]
+  );
```

Comment:
Prefer `useRoute()` when reading screen name.
Also, `banner_id` needs to be the trial ID associated to IT Wallet.
Adding typing is preferable, since it avoids to pass incorrect data.

```suggestion
  const route = useRoute();

  const trackBannerProperties: TrackITWalletBannerClosureProperties = React.useMemo(
    () => ({
      banner_id: itwTrialId,
      banner_page: route.name,
      banner_landing: "ITW_INTRO"
    }),
    [route]
  );
```